Aztec/Classes/Libxml2/DOMString.swift

```diff
@@ -493,23 +493,42 @@ extension Libxml2 {
             rootNode.replaceCharacters(in: range, with: descriptor)
         }
 
-        /// Applies horizontal ruler to the specified range.
+        /// Replaces the specified range with a Horizontal Ruler.
         ///
         /// - Parameters:
         ///     - range: the range to apply the style to.
         ///
-        func insertHorizontalRuler(at range: NSRange) {
+        func replaceWithHorizontalRuler(_ range: NSRange) {
             performAsyncUndoable { [weak self] in
-                self?.insertHorizontalRulerSynchronously(at: range)
+                self?.replaceSynchronouslyWithHorizontalRuler(range)
             }
         }
 
-        private func insertHorizontalRulerSynchronously(at range: NSRange) {
+        private func replaceSynchronouslyWithHorizontalRuler(_ range: NSRange) {
```

Comment:
To stick to the convention we're using in other methods, this should be `replaceWithHorizontalRuleSynchronously()`